src/BPEssentials/Enums/Chat.cs

```diff
@@ -4,6 +4,7 @@ public enum Chat
     {
         Disabled,
         Global,
-        StaffChat
+        StaffChat,
+        Muted
```

Comment:
Should we be using ExtendedPlayer#isMuted here instead of the Enum

Reply:
I dont think so?
It works...

Reply:
I shall ask R00T

Reply:
I think using the chat enum is better because then we don't need to create an extra property for it.